_docs/services/relational-database.md

```diff
@@ -117,38 +117,38 @@ In short, `cf bind-service` will provide a `DATABASE_URL` environment variable f
 
 The contents of the `DATABASE_URL` environment variable contain the credentials to access your database. Treat the contents of this and all other environment variables as sensitive.
 
-## Access the data in the database
+## Backups
+
+For shared plans (`shared-psql` and `shared-mysql`), RDS does not back up your data.
+
+For dedicated plans, RDS automatically retains daily backups for 14 days. These backups are AWS RDS storage volume snapshots, backing up the entire DB instance and not just individual databases. If you need to have a database restored using one of these backups, you can [email support](mailto:support@cloud.gov).  For non-emergency situations, please provide at least 48 hours advance notice.
 
-There are currently two ways to access the database directly.
+If you have an emergency situation, such as data loss or a compromised system, please [email support](mailto:support@cloud.gov) immediately and inform us of the situation.
 
-1. [The `cg-migrate-db` plugin](#cg-migrate-db-plugin). It is a self contained
-executable which will interactively assist with accessing the data in the
-database. It supports accessing data from different types of databases.
-1. [Manually accessing the database](#manually-access-a-database). This way
-requires manually downloading the tool(s) needed to access the database.
+When you do [contact support](mailto:support@cloud.gov) with a database backup request, please include the following information:
 
-### cg-migrate-db plugin
-You can access the data in your database via the `cg-migrate-db`
-plugin. See the [repository](https://github.com/18F/cg-migrate-db)
-for instructions on how to install the plugin, backup data, import data,
-download a local copy of the data, and upload a local copy of the data.
+- Your organization name
+- The space you are working within
+- The name of the application(s) connected to the database service you need a restoration performed on
```

Comment:
- phone numbers and contact information if it's an urgent situation.

Do _not_ share any passwords or details of any exploit or compromise. We'll call you if necessary, but we'll never ask you for a password over the phone.

Reply:
Excellent call - added!